src/askContainer/commands/listSkills.ts

```diff
@@ -13,6 +14,31 @@ export class ListSkillsCommand extends AbstractCommand<Array<SmapiResource<skill
         super('ask.container.listSkills');
     }
 
+    private async getSkillsList(
+        smapiClientInstance: skillManagementServiceClient, vendorId: string, nextToken: undefined | string, 
+        skillsList: Array<SmapiResource<skillSummary>>): Promise<Array<SmapiResource<skillSummary>>> {
+        const listSkills: model.v1.skill.ListSkillResponse = await smapiClientInstance.listSkillsForVendorV1(
+            vendorId, nextToken);
+        listSkills.skills?.forEach((listSkill) => {
+            let skillName = 'someSkill';
+            if (listSkill.nameByLocale !== undefined && Object.values(listSkill.nameByLocale).length > 0) {
+                if (listSkill.nameByLocale[EN_US_LOCALE]) {
+                    skillName = listSkill.nameByLocale[EN_US_LOCALE];
+                } else {
+                    skillName = Object.values(listSkill.nameByLocale)[0];
+                }
+            }
+            skillsList.push(new SmapiResource<skillSummary>(listSkill, skillName));
+        });
+
+        if (listSkills.isTruncated ?? true) {
```

Comment:
nit: why use `??` here ? Doesn't `if (listSkills.isTruncated)` work ?

Reply:
Because without that, the compiler complains about `truthy` checks as mentioned in this : https://github.com/typescript-eslint/typescript-eslint/blob/v3.10.1/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md